torch/profiler/utils.py

```diff
@@ -0,0 +1,49 @@
+from collections import deque
```

Comment:
We should name this `_utils.py` to avoid introducing symbols to the public namespace.

Reply:
Resolved